src/test/java/org/cactoos/http/HtTimedWireTest.java

```diff
@@ -76,24 +84,145 @@ public void worksFine() throws Exception {
         );
     }
 
-    // @todo #78:30min With #78 solved, HtWire stopped closing the connection
-    //  when send() is called and the following test stopped working for no
-    //  clear reason. Investigate the cause of this, fix it and unignore this.
-    @Ignore("see todo above")
-    // @checkstyle MagicNumberCheck (1 line)
-    @Test(expected = TimeoutException.class, timeout = 1000)
+    // @todo #87:30m For now I can't find proper OOP alternative for waiting.
+    //  Needs to create a new one. I think this can looks like this:
+    //  new Wait().seconds(long seconds). Needs to do this in the test
+    //  org.cactoos.http.HtTimedWireTest.failsAfterTimeoutCheckIfWait too
+    //  @checkstyle MagicNumberCheck (1 line)
+    @Test(timeout = 1000)
     public void failsAfterTimeout() throws Exception {
         // @checkstyle MagicNumberCheck (1 line)
         final long timeout = 100;
-        try (Socket blocker = new Socket()) {
-            blocker.connect(this.server.getLocalSocketAddress());
-            new HtTimedWire(
-                new HtWire(
-                    this.server.getInetAddress().getHostAddress(),
-                    this.server.getLocalPort()
-                ),
-                timeout
-            ).send(new InputOf("unused"));
+        final HtTimedWire wire = new HtTimedWire(
+            input -> {
+                TimeUnit.SECONDS.sleep(timeout + 1);
+                return input;
+            },
+            timeout
+        );
+        new Assertion<>(
+            HtTimedWireTest.TIMEOUT,
+            () -> wire.send(new InputOf(HtTimedWireTest.UNUSED)),
+            new TimeoutExceptionMatcher()
+        )
+            .affirm();
+    }
+
+    // @checkstyle MagicNumberCheck (1 line)
+    @Test(timeout = 1000)
+    public void failsAfterTimeoutCheckIfWait() throws Exception {
```

Comment:
@aivinog1 this test is a bit hard to follow. Is the objective here to verify that `HtTimedWire` actually failed after `timeout` expires and not before?

Reply:
No. Its primary objective from this comment: https://github.com/yegor256/cactoos-http/pull/92#discussion_r296634032. So this test verifies that execution of sending will be interrupted and we don't wait for executing of the origin in the HtTimedWire.